app/metrics.yaml

```diff
@@ -295,6 +295,20 @@ metrics:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-09-01"
+  top_sites_count:
+    type: string
```

Comment:
`counter` is a much better fit, except that you wouldn't be able to record `0`. Currently the proper way to do this is a `counter`+`boolean` flag to indicate absence of any top sites.

Reply:
@grigoryk updating this to use a counter + boolean is fine, but this also means that we should update the documentation for top_sites_count because no one would expect that this counter value can't be 0.

Fixing this in https://github.com/mozilla-mobile/fenix/pull/10752#issuecomment-631208869